internal/sdkprovider/service/kafka/kafka_schema.go

```diff
@@ -254,7 +257,7 @@ func resourceKafkaSchemaRead(_ context.Context, d *schema.ResourceData, m interf
 		}
 	} else {
 		// only update if was set to not empty values by the user
-		if _, ok := d.GetOk("compatibility_level"); ok || schemautil.IsImportingResource(d) {
+		if _, ok := d.GetOk("compatibility_level"); ok {
```

Comment:
Please leave a comment why do we do so ("only update if was set to not empty values by the user")

Reply:
please check again I simplified it further